<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveCaptureDialog</class>
 <widget class="QDialog" name="SaveCaptureDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save capture</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Choose part of capture to save</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="startTimeLabel">
         <property name="text">
          <string>(Start time)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QxtSpanSlider" name="simTimeSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="endTimeLabel">
         <property name="text">
          <string>(End time)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QxtSpanSlider</class>
   <extends>QSlider</extends>
   <header>libqxt/qxtspanslider.h</header>
   <slots>
    <signal>lowerValueChanged(int)</signal>
    <signal>upperValueChanged(int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SaveCaptureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SaveCaptureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simTimeSlider</sender>
   <signal>lowerValueChanged(int)</signal>
   <receiver>SaveCaptureDialog</receiver>
   <slot>simTimeSliderLowerValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simTimeSlider</sender>
   <signal>upperValueChanged(int)</signal>
   <receiver>SaveCaptureDialog</receiver>
   <slot>simTimeSliderUpperValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>simTimeSliderLowerValueChanged(int)</slot>
  <slot>simTimeSliderUpperValueChanged(int)</slot>
 </slots>
</ui>
